CONTRIBUTING.md

```diff
@@ -41,6 +42,10 @@ An agenda will be circulated in the gitter channel in advance of the meeting.
 Everyone is welcome.
 We look forward to meeting you there :hibiscus:
 
+## Contributing small documentation changes
+If you are new to GitHub and just have a small documentation change
+recommendation, please submit it to [our e-mail address](mailto:tedana.devs@gmail.com).
```

Comment:
Maybe add "and one of our developers will add it to the documentation directly."